src/frontend/src/icp/api/ckbtc-minter.api.ts

```diff
@@ -57,6 +53,20 @@ export const minterInfo = async ({
 	return getMinterInfo(rest);
 };
 
+export const getBtcAddress = async ({
```

Comment:
This should not have been part of this PR. I did not revert it before merging. It's a part of next PR and is not used yet, so not biggy and just for the record.